packages/react-data-grid/src/Cell.tsx

```diff
@@ -137,70 +124,28 @@ function Cell<R>({
     return allEvents;
   }
 
+  // FIXME: is this property used?
```

Comment:
I am not using it but not sure if others are using. I believe this is not a easily removed field.

Reply:
![image](https://user-images.githubusercontent.com/567105/67195909-be7c5d00-f3f1-11e9-81aa-43ed3ac94456.png)
It looks like that if a column is hidden, I think we should remove this feature, wdyt?

Reply:
What a weird API. Yeah, I think so, we should just remove it. I assumed it wrong that the columns will just be hidden but later realized that the cell is already rendered, and we only get an empty cell.

For this case it's same to putting `formatter` to return null

Reply:
Removed it now.